element_moseq/moseq_train.py

```diff
@@ -556,12 +590,14 @@ class FullFitTask(dj.Manual):
     """
 
     definition = """
-    -> PCAFit                                       # `PCAFit` Key
-    full_latent_dim              : int              # Latent dimension to use for the model full fitting
-    full_kappa                   : float            # Kappa value to use for the model full fitting
-    full_num_iterations          : int              # Number of Gibbs sampling iterations to run in the model full fitting
+    -> PCAFit                                           # `PCAFit` Key
+    full_latent_dim              : int                  # Latent dimension to use for the model full fitting
+    full_kappa                   : float                # Kappa value to use for the model full fitting
+    full_num_iterations          : int                  # Number of Gibbs sampling iterations to run in the model full fitting
     ---
-    full_fit_desc=''             : varchar(1000)    # User-defined description of the model full fitting task   
+    full_fit_desc=''             : varchar(1000)        # User-defined description of the model full fitting task   
+    task_mode='trigger'          :enum('trigger','load')# Trigger or load the task
+    model_name=''                : varchar(100)         # Name of the model to be loaded if `task_mode='load'`

```

Comment:
Is this required when the user sets `task_mode` to `load`? If so, I'd suggest making this attribute required too. We can do what we do for [processing_output_dir](https://github.com/datajoint/element-calcium-imaging/blob/main/element_calcium_imaging/imaging_no_curation.py#L235) where it is required but we can insert ` ' ' ` if it's not needed when `task_mode='trigger'`.

Reply:
This suggestion has been applied in [this commit](https://github.com/datajoint/element-moseq/pull/4/commits/c60d0f7d71321533707a114bbc66271506414495).